docs/concepts/assets/accounts.md

```diff
@@ -24,6 +24,53 @@ Any assets that are in a general account can be withdrawn or transferred.
 **[REST](../../api/rest/data-v2/trading-data-service-get-party.api.mdx)**: Use the API to check your public key for the accounts your assets are allocated to.
 :::
 
+## Vault
+A *vault* holds assets that have been contributed by one or more parties to be used by the vault manager for trading. The assets in a vault can only be used to enter and exit positions. Vaults have general and margin accounts associated with them.
+
+Contributors to a vault have a share of the value in the vault, based on how much their deposited amount contributed to its original value.
+
+The vault manager - the party that created the account - can charge fees on the performance and on management if they choose to. Vault managers can transfer the ownership of a vault to another user/public key.
+
+Each vault account handles one asset.
+
+### Create a vault
+Submitting a `CreateVault` transaction will create a vault for others to contribute to.
+
+The transaction needs to include the following information for the vault:
+- Settlement asset for the vault.
+- Metadata describing the vault.
+- Fee period, which sets the frequency of the vault's fees assessment
+- Management fee factor.
+- Performance fee factor.
+- Redemption dates
+- The cutoff period following the redemption date until which a redemption request is queued
+
+Once a vault is created, if the vault manager wants to transfer ownership to another party, they'll need to submit a `ChangeVaultOwnership` transaction with the vault ID and the Vega public key of the new vault manager.
```

Comment:
you may want to include something about update vault here too, including the rules for what can and cannot change - especially on redemption dates.